test/products.service.spec.ts

```
@@ -0,0 +1,349 @@
+    Object.assign(mockModelConstructor, mockProductModel);
+
+    mockInvoiceModel = {
+      aggregate: jest.fn().mockReturnThis(),
+      exec: jest.fn(),
+    };
+
+    mockTenantDb = {
+      model: jest.fn().mockImplementation((name) => {
+        if (name === 'Product') return mockModelConstructor;
+        if (name === 'Invoice') return mockInvoiceModel;
+        return null;
```

ESLint

[error] Use `undefined` instead of `null`.

(unicorn/no-null)